pkg/kube/secretcontroller/secretcontroller.go

```diff
@@ -219,28 +221,38 @@ func (c *Controller) addMemberCluster(secretName string, s *corev1.Secret) {
 		if _, ok := c.cs.remoteClusters[clusterID]; !ok {
 			if len(kubeConfig) == 0 {
 				log.Infof("Data '%s' in the secret %s in namespace %s is empty, and disregarded ",
-					clusterID, secretName, s.ObjectMeta.Namespace)
+					clusterID, secretName, s.Namespace)
 				continue
 			}
 
 			clientConfig, err := LoadKubeConfig(kubeConfig)
 			if err != nil {
 				log.Infof("Data '%s' in the secret %s in namespace %s is not a kubeconfig: %v",
-					clusterID, secretName, s.ObjectMeta.Namespace, err)
+					clusterID, secretName, s.Namespace, err)
+				continue
+			}
+
+			if err := ValidateClientConfig(*clientConfig); err != nil {
```

Comment:
Is client config validated in Galley? seems like it should always be validated.

Reply:
Yes, I checked

Reply:
How could it be ? The secret should never be read by Galley ( shouldn't even have permissions )

Reply:
I misunderstanding the question, @costinm is right, the multi cluster secret is never read by galley